src/chapters/data-and-variables/values-and-data-types.rst

```diff
@@ -92,35 +103,57 @@ JavaScript doesn't care whether you use single or double quotes to surround your
 More On Numbers
 ---------------
 
-When you type a large integer, you might be tempted to use commas between groups of three digits, as in ``42,000``. This is not a legal integer in JS, but it does mean something else, which is legal:
+When you type a large integer value, you might be tempted to use commas between groups of three digits, as in ``42,000``. This is not a legal integer in JavaScript, but it does mean something else, which is legal:
 
-.. code-block:: js
+.. sourcecode:: js
 
     console.log(42000);
     console.log(42,000);
 
+**Output**
+
+::
+
+   42000
+   42 0
 
-Well, that's not what we expected at all! Because of the comma, JS chose to treat this as a *pair* of values. In fact, the ``console.log()`` function can print any number of values as long as you separate them by commas. Notice that the values are separated by spaces when they are displayed.
+Well, that's not what we expected at all! Because of the comma, JavaScript chose to treat ``42,000`` as a *pair* of values. In fact, the ``console.log()`` function can print any number of values as long as you separate them by commas. Notice that the values are separated by spaces when they are displayed.
 
-.. code-block:: js
+.. sourcecode:: js
 
     console.log(42, 17, 56, 34, 11, 4.35, 32);
     console.log(3.4, "hello", 45);
 
-Remember not to put commas or spaces in your integers, no matter how big they are. Also revisit what we said in the previous chapter: formal languages are strict, the notation is concise, and even the smallest change might mean something quite different from what you intended.
+**Output**
+
+::
+
+   42 17 56 34 11 4.35 32
+   3.4 'hello' 45
+
+Remember not to put commas or spaces in your integers, no matter how big they are. Also revisit what we said in the previous chapter: formal languages are strict, the notation is concise, and even the smallest change might mean something quite different from what you intend.
```

Comment:
Be aware that if you change Chapter 2, this sentence may not make sense anymore.

Reply:
I replaced "previous chapter" with a reference, which will stay up-to-date even after reordering.